server/controllers/tracking.js

```diff
@@ -0,0 +1,20 @@
+const axios = require('axios');
+const config = require('../../config.js');
+
+module.exports = {
+  postInteraction: (req, res) => {
+    var data = {element: req.body.element, widget: req.body.widget, time: req.body.time}
+
+    axios({
+      method: 'post',
+      url: `https://app-hrsei-api.herokuapp.com/api/fec2/hr-rpp/interactions`,
```

Comment:
Aha! So that's what we do with the interactions... didn't realize there was an interactions API :D